tests/unit_tests/util.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2023-2023, The Monero Project
+// Copyright (c) 2024-2023, The Monero Project
```

Comment:
```suggestion
// Copyright (c) 2023-2024, The Monero Project
```

Reply:
Great spot! Need to see where the parser failed

Reply:
same here, simply "same year-same year" definitely needs to be handled by the parser as this is valid data

https://github.com/monero-project/monero/commit/d9c7cd5a89cd1baa270a097bae1a2eff7bd4b2e9#diff-fa65a3bd0cd6bc935c215678fa298a75e253cd18649150e92ad28042f91a2986R1

Reply:
merging this as is should be ok. then ill make a PR modifying the typos with hinto as author, with a 2nd commit fixing copyCat to account for sameyear-sameyear as shown here https://github.com/monero-project/monero/compare/master...plowsof:monero:master